app/services/chat.ts

```diff
@@ -94,8 +94,9 @@ export class ChatService extends Service {
         nodeIntegration: false,
       },
     });
-
-    this.navigateToChat();
+    this.windowsService.updateStyleBlockers('main', true);
+    await this.navigateToChat();
+    this.windowsService.updateStyleBlockers('main', false);
```

Comment:
What's the purpose of disabling style blockers here?  Is there a change this raises an exception and we get stuck with style blockers hidden?  Can this take a long time (servers time out after 60 seconds of loading) and we are stuck with style blockers hidden for that all that time?  This change makes me nervous.

Reply:
`navigateToChat` appears to catch errors from what i saw in the codebase, but the reason for this is rendering the browserWindow in chat while performing this navigation is what is causing the crashy state

Reply:
It only catches 1 very specific type of error that happens when the page redirects, which is expected.  I think that style blockers is not the right thing to use here, and it needlessly hides many other things.  We should just hide the chat during this time.

Reply:
i asked if that was a viable fix in FE chat but got no responses 😅 will make that change